src/Advection/centered_reconstruction.jl

```diff
@@ -46,7 +46,7 @@ function Centered(FT::DataType = Float64; grid = nothing, order = 2)
 
     N  = Int(order ÷ 2)
     if N > 1 
-        coefficients    = Tuple(nothing for i in 1:6)
+        coefficients  = Tuple(nothing for i in 1:6)
         # coefficients = compute_reconstruction_coefficients(grid, FT, :Centered; order)
```

Comment:
These are the `stretched` weights for centered reconstruction. There were commented out because I found out that for centered reconstruction using stretched weights makes it more unstable. We could think about removing them entirely (in another PR)